### PR TITLE
Simplify output of azure ci for unbuilded wheel

### PR DESCRIPTION
**Pull request by alkino**
On master only the wheel for python 3.9 is generated, so disable matrix for
other versions of python.
During the nightly generate wheel for all versions.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -181,22 +184,27 @@ jobs:
...
+        Python37:
+          python.version: '3.7'
+          python.org.version: '3.7.7'
+        Python38:
+          python.version: '3.8'
+          python.org.version: '3.8.2'
+        Python39:
+          python.version: '3.9'
+          python.org.version: '3.9.0'
+      ${{ if ne(variables.buildWheel, True) }}:
```
Comment (ohm314) [inline azure-pipelines.yml:200]:
```suggestion
      ${{ if eq(variables.buildWheel, False) }}:
```
maybe just a matter of taste.. but this seems a bit more readable

**Review by ohm314 — APPROVED** [review]
Really cool! Thanks for figuring this out :)

**Comment by alkino** [comment]
Please retest

**Comment by alkino** [comment]
Please retest

**Pull request merged**